docs/how-to/h-deploy/h-deploy-aws.md

```diff
@@ -111,7 +111,7 @@ Deploy and integrate Kafka and ZooKeeper:
 
 ```shell
 juju deploy zookeeper -n3 --channel 3/stable [--constraints "instance-type=<INSTANCE_TYPE>"] 
-juju deploy kafka -n3 --channel 3/stable [--constraints "instance-type=<INSTANCE_TYPE>"]
+juju deploy kafka -n3 --trust --channel 3/stable [--constraints "instance-type=<INSTANCE_TYPE>"]
```

Comment:
I already deleted this change on [Discourse](https://discourse.charmhub.io/t/charmed-kafka-documentation-how-to-deploy-on-aws/15881), but it seems like Discourse API still serves the old version for some reason. So it still exists on [Canonical.com](https://canonical.com/data/docs/kafka/iaas/h-deploy-aws#p-34315-deploy-charms) and here. Let's give it a time.

Reply:
It got updated on Canonical.com, but still not in this PR